lib/main_common.pm

```diff
@@ -2124,6 +2124,9 @@ sub load_publiccloud_tests {
     elsif (get_var('PUBLIC_CLOUD_LTP')) {
         loadtest 'publiccloud/run_ltp';
     }
+    elsif (get_var('PUBLIC_CLOUD_ACCNET') && get_required_var('PUBLIC_CLOUD_PROVIDER', 'AZURE')) {
```

Comment:
I think `get_required_var()` don't have a default value.
I think you mean something like `get_required_var('PUBLIC_CLOUD_PROVIDER') == 'AZURE'`

Reply:
ok, you are right

Reply:
would prefer to run it if it is not azure and got failure , instead allowing to add this variable to any random test without any harm

Reply:
you didn't get my point here. Totally agree - there is no point of running this feature on clouds that we know it will fail. But there is no point to define this variable for test suite which will not run for Azure. With your approach you can add variable even on Medium level  so all tests will have it and it will not harm , and with mine you need to insert only exactly to proper test case otherwise you will get red bubbles. 
And this is the point , SLENKINS_TESTSUITES_REPO is good example of what I am trying to avoid

Reply:
what is your suggestion then? to remove PUBLIC_CLOUD_ACCNET variable and run this test if check_var('PUBLIC_CLOUD_PROVIDER', 'AZURE')  ?

Reply:
Don't know if it will work. Cause then all tests azure testsuites will add this testmodule. And this means it is also added to upload_img for example.